### ISSUE TITLE
Increased valine in urine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6048]
**Preferred term label:** Increased valine in urine

**Synonyms** Valinuria

**Definition (free text, please give PubMed ID)** Concentration of valine in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Neutral hyperaminoaciduria HP:0008353

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding as HP:0033088
diseases - 
https://www.omim.org/clinicalSynopsis/277100

**Comment by pnrobinson** [comment]
added
